### PR TITLE
Handling Image handler

### PR DESCRIPTION
**Pull request by hemanthkatam**
Handling images with quill-image-uploader. 

with this it will add images in url format.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,7 +9,11 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.js"></script>
     <script src="https://cdn.quilljs.com/1.3.6/quill.js"></script>
+    <script src="https://cdn.quilljs.com/1.3.6/quill.min.js"></script>
```
Comment (KillerCodeMonkey) [inline src/index.html:12]: quill is already added one line above

File: src/index.html [hunk 2]
```diff
@@ -33,8 +37,9 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css" />
     <link rel="stylesheet" href="https://cdn.quilljs.com/1.3.6/quill.bubble.css" />
     <link rel="stylesheet" href="https://cdn.quilljs.com/1.3.6/quill.snow.css" />
+    <link href="https://cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet" />
+  <link href="https://unpkg.com/quill-image-uploader@1.2.1/dist/quill.imageUploader.min.css" rel="stylesheet" />
 
-
```
Comment (KillerCodeMonkey) [inline src/index.html:43]: remove unnecessary new lines

File: src/index.html [hunk 3]
```diff
@@ -33,8 +37,9 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css" />
     <link rel="stylesheet" href="https://cdn.quilljs.com/1.3.6/quill.bubble.css" />
     <link rel="stylesheet" href="https://cdn.quilljs.com/1.3.6/quill.snow.css" />
+    <link href="https://cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet" />
+  <link href="https://unpkg.com/quill-image-uploader@1.2.1/dist/quill.imageUploader.min.css" rel="stylesheet" />
```
Comment (KillerCodeMonkey) [inline src/index.html:41]: fix indention

**Review by KillerCodeMonkey — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (hemanthkatam) [reply src/index.html]: resolved

File: src/index.html (hunk 3 above)
Comment (hemanthkatam) [reply src/index.html]: resolved

File: src/index.html (hunk 2 above)
Comment (hemanthkatam) [reply src/index.html:41]: resolved

**Review by hemanthkatam — COMMENTED** [review]
resolved

**Pull request merged**